mswindows/GRASS-Installer.nsi.tmpl

```diff
@@ -891,27 +829,13 @@ FunctionEnd
 Section "Important Microsoft Runtime DLLs" SecMSRuntime
 
 	;Set the size (in KB)  of the archive file
-	StrCpy $ARCHIVE_SIZE_KB 18512
+	StrCpy $ARCHIVE_SIZE_KB 833
 	
 	;Set the size (in KB) of the unpacked archive file
 	AddSize 13500
 	
-	StrCpy $HTTP_PATH "http://download.osgeo.org/osgeo4w/${PLATFORM}/release/msvcrt/"
-        !if ${PLATFORM} == "x86_64"
-            StrCpy $ARCHIVE_NAME "msvcrt-1.0.1-9.tar.bz2"
-            StrCpy $ARCHIVE_NAME2008 "msvcrt2008-1.0.0-1.tar.bz2"
-            StrCpy $ARCHIVE_NAME2010 "msvcrt2010-1.0.0-1.tar.bz2"
-            StrCpy $ARCHIVE_NAME2012 "msvcrt2012-1.0.0-1.tar.bz2"
-            StrCpy $ARCHIVE_NAME2013 "msvcrt2013-1.0.0-1.tar.bz2"
-            StrCpy $ARCHIVE_NAME2015 "msvcrt2015-1.0-1.tar.bz2"
-        !else
-            StrCpy $ARCHIVE_NAME "msvcrt-1.0.1-14.tar.bz2"
-            ;StrCpy $ARCHIVE_NAME2005 "msvcrt2005-1.0-1.tar.bz2"
-            StrCpy $ARCHIVE_NAME2008 "msvcrt2008-1.0-1.tar.bz2"
-            StrCpy $ARCHIVE_NAME2010 "msvcrt2010-1.0-1.tar.bz2"
-            StrCpy $ARCHIVE_NAME2013 "msvcrt2013-1.0-1.tar.bz2"
-            StrCpy $ARCHIVE_NAME2015 "msvcrt2015-1.0-1.tar.bz2"
-        !endif
+	StrCpy $HTTP_PATH "http://download.osgeo.org/osgeo4w/v2/${PLATFORM}/release/msvcrt2019/"
```

Comment:
`msvcrt2019-14.2-1.tar.bz2` (http://download.osgeo.org/osgeo4w/v2/x86_64/release/msvcrt2019/msvcrt2019-14.2-1.tar.bz2) contains only `dll` files. If we don't remove them in `GRASS-Packager.bat.tmpl` then there is no need to download the archive by installer and copy its content into target directory. @hellik What do you think?

Reply:
I can't remember exactly, but isn't the download due to license reason? I think we have a small paragraph why we download it extra, right? Lets have a check there first.

Reply:
Let's leave for separated PR if needed. What do you think @hellik ?

Reply:
yes, lets keep it as it is at the moment. and open a separated PR if when we have checked the licence questions.